### PR TITLE
Ginkgo: Fix ubuntu repos

### PR DESCRIPTION
**Pull request by eloycoto**
Ubuntu 17.04 is EOL and the ubuntu repos are gone. This commit changes
the repositories meanwhile ubuntu 17.10 image is not in place.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Comment by eloycoto** [comment]
test-all-ginkgo

**Comment by eloycoto** [comment]
test-me-please

File: test/provision/k8s_install.sh [hunk 1]
```diff
@@ -58,7 +70,6 @@ esac
 
 retry_function "apt-get update"
 retry_function "apt-get install --allow-downgrades -y \
-    llvm \
```
Comment (joestringer) [inline test/provision/k8s_install.sh:61]: I think we need recent llvm, should we also add repos from here?

https://apt.llvm.org/

**Review by joestringer — COMMENTED** [review]
If CI passes, LGTM. We can revert when the image is updated.

**Comment by joestringer** [comment]
Fixes: #2562

File: test/provision/k8s_install.sh (hunk 1 above)
Comment (eloycoto) [reply test/provision/k8s_install.sh:61]: It's already installed  on the base image ;-)

**Comment by ianvernon** [comment]
The CI got to the stages where the tests are ran, indicating that this is fixed. LGTM.

**Review by ianvernon — APPROVED** [review]

**Pull request merged**